include/boost/di.hpp

```diff
@@ -268,7 +268,7 @@ struct make_index_sequence_impl<10> : index_sequence<0, 1, 2, 3, 4, 5, 6, 7, 8,
 #endif
 template <int N>
 using make_index_sequence = typename make_index_sequence_impl<N>::type;
-}  // namespace aux
+}
```

Comment:
Format was reverted to fix travis.
Got broken at https://github.com/boost-experimental/di/pull/345